### ISSUE TITLE
Rankings have H1 tag unecessarily

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#5502]
## Problem/Motivation
When reviewing https://www.arizona.edu/about/rankings, the rankings were discovered to have the H1 tag.

### Describe the bug
The tag should be selectable, while the style should remain the same.

### To Reproduce
Create content with a ranking paragraph type.

## Proposed resolution
Change the ranking heading level, style, and add a heading level selection.

### Expected behavior
I expect the ranking to have proper headings.

##  Screenshots
<img width="1747" height="1251" alt="Image" src="https://github.com/user-attachments/assets/004ba02e-0b3a-4709-8f64-700140e24806" />

**Comment by danahertzberg** [comment]
Follow up to make heading level selectable
